### PR TITLE
Added some bold, as well as changed some sentences

### PR DESCRIPTION
**Pull request by Jette94**

### CODE REVIEW

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In this article, we'll take a look at the basic steps of creating a Form and add
 
 You can manage the Forms in the **Forms** section of the Umbraco backoffice. You need to have access to the section in order to see it.
 
-If you do not see the **Forms** section, you will need to either log in with an Administrator account or request access from someone with Administrator permissions for your site. An Administrator can give permission to view the Forms section to your individual account or the user group your account belongs to from within the Users section of the backoffice.
+If you do not see the **Forms** section, you will need to either log in with an Administrator account or request access from someone with Administrator permissions for your site. An Administrator can give permission to view the **Forms** section to your individual account or the user group that your account belongs to from within the **Users** section of the backoffice.
```
Comment (sofietoft) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/index.md:14]:
```suggestion
If you do not see the **Forms** section, you might need to request access from the site Administrator. An Administrator can give permission to view the **Forms** section. This is done from within the **Users** section of the backoffice.
```

**Review by sofietoft — COMMENTED** [review]
Thanks a bunch for the PR @Jette94 ! 💪 

I've made a suggestion to shorten a couple of sentences.
It removed quite a bit of info, but I think the info is kinda redundant in this article anyways 🤔 
What do you think?

**Comment by Jette94** [comment]
> Thanks a bunch for the PR @Jette94 ! 💪
> 
> I've made a suggestion to shorten a couple of sentences. It removed quite a bit of info, but I think the info is kinda redundant in this article anyways 🤔 What do you think?

Looking good, Sofie :D

**Comment by sofietoft** [comment]
Great @Jette94 !
Let's get this one merged in then 😄 👏

**Pull request merged**